docs/framework/unmanaged-api/profiling/icorprofilercallback9-dynamicmethodunloaded-method.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "ICorProfilerCallback9::DynamicMethodUnloaded Method"
+ms.custom: ""
+ms.date: "04/10/2018"
+ms.prod: ".net-framework"
+ms.technology: 
+  - "dotnet-clr"
+ms.topic: "article"
+api_name: 
+  - "ICorProfilerCallback9.DynamicMethodUnloaded"
+api_location: 
+  - "mscorwks.dll"
+  - "corprof.idl"
+api_type: 
+  - "COM"
+author: "rpetrusha"
+ms.author: "ronpet"
+manager: "wpickett"
+ms.workload: 
+  - "dotnet"
+---
+# ICorProfilerCallback9::DynamicMethodUnloaded Method
+[Supported in the .NET Framework 4.7.2 and later versions]  
```

Comment:
not related to your PR: I just noticed now that we repeat this info twice in every topic: one here and one on .NET Framework versions

Reply:
Makes sense.